backend/src/test/java/backend/mulkkam/cup/service/CupServiceIntegrationTest.java

```diff
@@ -188,4 +193,149 @@ void success_withExistedMemberId() {
             });
         }
     }
+
+    @DisplayName("컵을 수정할 때에")
+    @Nested
+    class Modify {
+
+        @DisplayName("컵 이름 및 용량이 수정된다")
+        @Test
+        void success_withValidData() {
+            // given
+            Member member = new MemberFixture().build();
+            memberRepository.save(member);
+
+            String beforeCupNickName = "변경 전";
+            String afterCupNickName = "변경 후";
+            Integer beforeCupAmount = 500;
+            Integer afterCupAmount = 1000;
+
+            Cup cup = new CupFixture()
+                    .member(member)
+                    .cupAmount(new CupAmount(beforeCupAmount))
+                    .cupNickname(new CupNickname(beforeCupNickName))
+                    .build();
+
+            Cup savedCup = cupRepository.save(cup);
+            CupNicknameAndAmountModifyRequest cupNicknameAndAmountModifyRequest = new CupNicknameAndAmountModifyRequest(
+                    afterCupNickName,
+                    afterCupAmount
+            );
+
+            // when
+            cupService.modifyNicknameAndAmount(
+                    savedCup.getId(),
+                    member.getId(),
+                    cupNicknameAndAmountModifyRequest
+            );
+
+            Cup changedCup = cupRepository.findById(savedCup.getId())
+                    .orElseThrow();
+
+            // then
+            assertSoftly(softly -> {
+                softly.assertThat(changedCup.getCupAmount().value()).isEqualTo(afterCupAmount);
+                softly.assertThat(changedCup.getNickname().value()).isEqualTo(afterCupNickName);
+            });
+        }
+
+        @DisplayName("수정할 컵만 변경된다")
+        @Test
+        void success_onlyOneCupShouldBeModified() {
+            // given
+            Member member = new MemberFixture().build();
+            memberRepository.save(member);
+
+            String beforeCupNickName1 = "변경 전1";
+            String beforeCupNickName2 = "변경 전2";
+            String afterCupNickName = "변경 후";
+            Integer beforeCupAmount1 = 300;
+            Integer beforeCupAmount2 = 500;
+            Integer afterCupAmount = 1000;
+
+            Cup cup1 = new CupFixture()
+                    .member(member)
+                    .cupNickname(new CupNickname(beforeCupNickName1))
+                    .cupAmount(new CupAmount(beforeCupAmount1))
+                    .build();
+
+            Cup cup2 = new CupFixture()
+                    .member(member)
+                    .cupNickname(new CupNickname(beforeCupNickName2))
+                    .cupAmount(new CupAmount(beforeCupAmount2))
+                    .build();
+
+            cupRepository.saveAll(List.of(
+                    cup1,
+                    cup2
+            ));
+
+            CupNicknameAndAmountModifyRequest cupNicknameAndAmountModifyRequest = new CupNicknameAndAmountModifyRequest(
+                    afterCupNickName,
+                    afterCupAmount
+            );
```

Comment:
```suggestion
            String beforeCupNickName1 = "변경 전1";
            Integer beforeCupAmount1 = 300;
            Cup cup1 = new CupFixture()
                    .member(member)
                    .cupNickname(new CupNickname(beforeCupNickName1))
                    .cupAmount(new CupAmount(beforeCupAmount1))
                    .build();

            String beforeCupNickName2 = "변경 전2";
            Integer beforeCupAmount2 = 500;
            Cup cup2 = new CupFixture()
                    .member(member)
                    .cupNickname(new CupNickname(beforeCupNickName2))
                    .cupAmount(new CupAmount(beforeCupAmount2))
                    .build();

            cupRepository.saveAll(List.of(
                    cup1,
                    cup2
            ));

            String afterCupNickName = "변경 후";
            Integer afterCupAmount = 1000;
            CupNicknameAndAmountModifyRequest cupNicknameAndAmountModifyRequest = new CupNicknameAndAmountModifyRequest(
                    afterCupNickName,
                    afterCupAmount
            );
```

이런 식으로 변수를 사용하는 위치에 가깝게 선언하는 방식은 어떻게 생각하시나요? 
_변수는 사용하는 곳에 가깝게 둬라_
가라사대를 인용해봅니다 ㅎ.ㅎ